### ISSUE TITLE
Increase timeline avatar size to 32x32px

### ISSUE THREAD

**Issue by ShadowJonathan** [vector-im/element-web#17174]
![image](https://user-images.githubusercontent.com/22740616/117141665-1a79bd00-adaf-11eb-80c3-702be6d2f0a4.png)

Above image contains my avatar shown two times, the first time is with the `<img>` element expanded to 32x32px, and the second (at the bottom) is the 30x30px default.

Currently, conduit, synapse, dendrite, and the media repo all send a 32x32px image when requested to send a 30x30px one, this results in that *slight* blur you see right there (on the bottom image)

Then, changing the default avatar (request and frame) size to 32x32px seems like a sensible change, imo.

**Comment by t3chguy** [comment]
FTR this isn't an issue with HiDPI as scaling a 96px -> 30px@2x is pretty alright
![image](https://user-images.githubusercontent.com/2403652/117142180-514bd500-ada7-11eb-9f15-1d85f032f1e2.png)
![image](https://user-images.githubusercontent.com/2403652/117142204-5741b600-ada7-11eb-97d4-b73f41f0e29e.png)

**Comment by ShadowJonathan** [comment]
Still, on 1:1 sized screens, this change would make a subtle but nice difference, and i dont think it's that too much to ask.

**Comment by t3chguy** [comment]
I mean, its a strange fix given that the next up and coming server might not bin at 32px and instead have one at 30px...

**Comment by ShadowJonathan** [comment]
FTR, this is Synapse's, Dendrite's, and Media Repo's default behaviour as well, they all crop at 32px and 96px, but dont adhere dynamic scaling requests (synapse does if you manually enable it)

**Comment by t3chguy** [comment]
Sure, but the spec doesn't mandate servers must offer 32px thumbs; it only suggests such. 

```
Servers SHOULD produce thumbnails with the following dimensions and methods:

32x32, crop
96x96, crop
320x240, scale
640x480, scale
800x600, scale

**Comment by ShadowJonathan** [comment]
Yes, but the reality is is that the servers do not respond with any other thumbnail except those sizes. Element Web currently scales the avatar frame to 30x30px, and that causes the blurry look on most screens.

**Comment by chronos1970** [comment]
#19540 might also be relevant
+1 for fixing this. While I don't think it's a serious usability issue, I believe it contributes to making the user experience feel less polished.
